src/system/components/handler.clj

```diff
@@ -1,7 +1,8 @@
 (ns system.components.handler
   (:require [com.stuartsierra.component :as component]
             [lang-utils.core :refer [contains+? ∘]]
-            [compojure.core :as compojure]))
+            [compojure.core :as compojure]
+            [bidi.ring :as bidi]))
```

Comment:
Yes, this is hard to argue against. 
Please feel free to submit PR if you feel strongly about this.